### ISSUE TITLE
Error parsing segments

### ISSUE THREAD

**Issue by dchertousov** [Travelport-Ukraine/uapi-json#357]
With segments in terminal like this:

```
 1. LH 6331 Z  26DEC VIEFRA TK1  0710   0840  O*       E WE  1
         OPERATED BY AUSTRIAN AIRLINES
 2. LH  484 Z  26DEC FRAPTY HK1  1040   1640  O*       E WE  1
         OPERATED BY LUFTHANSA CITYLINE GMBH
 3. CM  192 D  26DEC PTYSJO TK1  1841   1906  O*         WE
 4. HTL ZZ AK1  SJO 26DEC-OUT27DEC /H-HAMPTON INN/R-AK1/BC-B/CF-
    GO14626976/RQ-EUR125.00/SI-SINGLE
 5. RZ 1080 Y  27DEC SJOTMU AK1  0710   0740  N          TH
 6. RZ 1081 Y  10JAN TMUSJO AK1  0750   0820  N          TH
 7. CM  163 D  10JAN SJOPTY HK1  1129   1352  O*         TH
 8. CM  268 D  10JAN PTYSDQ TK1  1550   1917  O*         TH
 9. HTL ZZ AK1  HEX 10JAN-OUT12JAN /H-BILLINI/R-AK1/BC-B/CF-GO14
    631374/RQ-EUR611.00/SI-EXECUTIVE SUITE BFST INCL
10. UA 1985 Z  12JAN SDQEWR HK1  1228   1535  O*       E SA
11. OS   90 Z  15JAN EWRVIE TK1  1755  #0820  O*         TU
```

The parsed segments response is as following:

```json
[
    {
      "from": "VIE",
      "to": "FRA",
      "group": 0,
      "departure": "2018-12-26T07:10:00.000+01:00",
      "arrival": "2018-12-26T08:40:00.000+01:00",
      "airline": "LH",
      "flightNumber": "6331",
      "uapi_segment_ref": "ygSaOeBAAA/BZWpr7HAAAA==",
      "serviceClass": "Business",
      "plane": [
        "320"
      ],
      "duration": [
        "90"
      ],
      "techStops": [],
      "index": 1,
      "status": "TK",
      "bookingClass": "Z"
    },
    {
      "from": "FRA",
      "to": "PTY",
      "group": 0,
      "departure": "2018-12-26T10:40:00.000+01:00",
      "arrival": "2018-12-26T16:40:00.000-05:00",
      "airline": "LH",
      "flightNumber": "484",
      "uapi_segment_ref": "ygSaOeBAAA/BcWpr7HAAAA==",
      "serviceClass": "Business",
      "plane": [
        "343"
      ],
      "duration": [
        "720"
      ],
      "techStops": [],
      "index": 2,
      "status": "HK",
      "bookingClass": "Z"
    },
    {
      "from": "PTY",
      "to": "SJO",
      "group": 0,
      "departure": "2018-12-26T18:41:00.000-05:00",
      "arrival": "2018-12-26T19:06:00.000-06:00",
      "airline": "CM",
      "flightNumber": "192",
      "uapi_segment_ref": "ygSaOeBAAA/B7Wpr7HAAAA==",
      "serviceClass": "Business",
      "plane": [
        "73G"
      ],
      "duration": [
        "85"
      ],
      "techStops": [],
      "index": 3,
      "status": "TK",
      "bookingClass": "D"
    },
    {
      "from": "SJO",
      "to": "TMU",
      "group": 1,
      "departure": "2018-12-27T07:10:00.000-06:00",
      "arrival": "2018-12-27T07:40:00.000-06:00",
      "airline": "RZ",
      "flightNumber": "1080",
      "uapi_segment_ref": "ygSaOeBAAA/B9Wpr7HAAAA==",
      "serviceClass": "Economy",
      "plane": [
        "CNC"
      ],
      "duration": [
        "30"
      ],
      "techStops": [],
      "index": 5,
      "status": "AK",
      "bookingClass": "Y"
    },
    {
      "from": "TMU",
      "to": "SJO",
      "group": 2,
      "departure": "2019-01-10T07:50:00.000-06:00",
      "arrival": "2019-01-10T08:20:00.000-06:00",
      "airline": "RZ",
      "flightNumber": "1081",
      "uapi_segment_ref": "ygSaOeBAAA/BBXpr7HAAAA==",
      "serviceClass": "Economy",
      "plane": [
        "CNC"
      ],
      "duration": [
        "30"
      ],
      "techStops": [],
      "index": 6,
      "status": "AK",
      "bookingClass": "Y"
    },
    {
      "from": "SJO",
      "to": "PTY",
      "group": 2,
      "departure": "2019-01-10T11:29:00.000-06:00",
      "arrival": "2019-01-10T13:52:00.000-05:00",
      "airline": "CM",
      "flightNumber": "163",
      "uapi_segment_ref": "ygSaOeBAAA/BFXpr7HAAAA==",
      "serviceClass": "Business",
      "plane": [
        "738"
      ],
      "duration": [
        "83"
      ],
      "techStops": [],
      "index": 7,
      "status": "HK",
      "bookingClass": "D"
    },
    {
      "from": "PTY",
      "to": "SDQ",
      "group": 2,
      "departure": "2019-01-10T15:50:00.000-05:00",
      "arrival": "2019-01-10T19:17:00.000-04:00",
      "airline": "CM",
      "flightNumber": "268",
      "uapi_segment_ref": "ygSaOeBAAA/BHXpr7HAAAA==",
      "serviceClass": "Business",
      "plane": [
        "738"
      ],
      "duration": [
        "147"
      ],
      "techStops": [],
      "index": 8,
      "status": "TK",
      "bookingClass": "D"
    }
  ]
```

**Comment by dchertousov** [comment]
Two segments are mssing

**Comment by dchertousov** [comment]
Closed by #358
